docs/reference/nuget-config-file.md

```diff
@@ -35,6 +35,7 @@ Contains miscellaneous configuration settings, which can be set using the [`nuge
 | repositoryPath (`packages.config` only) | The location in which to install NuGet packages instead of the default `$(Solutiondir)/packages` folder. A relative path can be used in project-specific `nuget.config` files. |
 | defaultPushSource | Identifies the URL or path of the package source that should be used as the default if no other package sources are found for an operation. |
 | http_proxy http_proxy.user http_proxy.password no_proxy | Proxy settings to use when connecting to package sources; `http_proxy` should be in the format `http://<username>:<password>@<domain>`. Passwords are encrypted and cannot be added manually. For `no_proxy`, the value is a comma-separated list of domains the bypass the proxy server. You can alternately use the http_proxy and no_proxy environment variables for those values. For additional details, see [NuGet proxy settings](http://skolima.blogspot.com/2012/07/nuget-proxy-settings.html) (skolima.blogspot.com). |
+| maxHttpRequestsPerSource | Controls the maximum number of parallel requests sent from NuGet to every package source for package dependency resolution and downloads. The default value on `.NET Core` is `Int32.MaxValue` which is derived from `HttpClientHandler.MaxConnectionsPerServer` property. This setting has no impact on `.NET Core` for `Mac OS` because the throttling limit is set to `16` to avoid too many open files error. `NuGet client tools` that runs on `.NET Framework` such as `Visual Studio` and `nuget.exe` sets `ServicePointManager.DefaultConnectionLimit` property to `64` on `Windows` and to `1` on `Mono` to control maximum number of concurrent connections allowed for every package source. Hence configuring `maxHttpRequestsPerSource` property to a value less than `64` could impact performance of `Visual Studio` and `nuget.exe` on `Windows`. Maximum degree of parallelism for `Packages.config` style projects is set to `Environment.ProcessorCount` by default. Hence configuring `maxHttpRequestsPerSource` property to a value less than the default could impact performance of `Packages.config` style projects. |
```

Comment:
I think we might be adding a bit too much about the techgnical implementation by mentioning service point manager and the likes. 

.NET Core is also a term we probably shouldn't use as much going forward, but really just talk in the terms of tool like you did in the later part. 

Finally, the description is on the richer side. Call out the defaults and just say, lowering the parallelization may worsen performance.

Reply:
updated text in the latest commit. Please review again and let me know the feedback.